templates/centos6.tpl.yaml

```diff
@@ -1,7 +1,10 @@
+# Change the version variable to the next released version of common-templates
+# if a non-backward compatible change was made to the template
```

Comment:
Well, there should be no incompatible changes to the template :) In case you need to do such change, create a new template with the same name, labels and new version.

Reply:
I will remove this comment for now until I make sure the original flow is supported by the UI team and then I will prepare a new PR with comments and documentation